src/TabCompleteEntries.js

```diff
@@ -113,8 +113,26 @@ class MemberEntry extends Entry {
     }
 }
 
-MemberEntry.fromMemberList = function(members) {
+MemberEntry.fromMemberList = function(room, members) {
+    // build up a dict of when, in the history we have cached,
+    // each member last spoke
+    const lastSpoke = {};
+    const timelineEvents = room.getLiveTimeline().getEvents();
```

Comment:
We now depend on the events in the room to determine ordering, but we don't call `_updateTabCompleteList` when we get incoming events. We probably should.